Jobs/Notifications/OpenAccessMailUsers.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+/**
+ * @file Jobs/Notifications/OpenAccessMailUsers.php
+ *
+ * Copyright (c) 2014-2022 Simon Fraser University
+ * Copyright (c) 2000-2022 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class OpenAccessMailUsers
+ * @ingroup jobs
+ *
+ * @brief Class to send notification about statistical report to editors
+ */
+
+namespace APP\Jobs\Notifications;
+
+use APP\core\Application;
+use APP\facades\Repo;
+use APP\journal\Journal;
+use APP\mail\mailables\OpenAccessNotify;
+use APP\template\TemplateManager;
+use Illuminate\Bus\Batchable;
+use Illuminate\Support\Facades\Mail;
+use Illuminate\Support\LazyCollection;
+use PKP\Support\Jobs\BaseJob;
+use Symfony\Component\Mime\Message;
+
+class OpenAccessMailUsers extends BaseJob
+{
+    use Batchable;
+
+    protected LazyCollection $users;
+    protected int $journalId;
```

Comment:
We are trying to use `$contextId` for any new code and slowly phasing out the use of `$journalId`.